src/ophys_etl/transforms/classification.py

```diff
@@ -0,0 +1,304 @@
+                                  "the (height, width) dimensions. Please "
+                                  "check the input data.")
+        mat = coo_matrix(arr)
+        data.update({"coo_roi": mat})
+        return data
+
+
+class InferenceInputSchema(ArgSchema):
+    """ Argschema parser for module as a script """
+    neuropil_traces_path = H5InputFile(
+        required=True,
+        description=(
+            "Path to neuropil traces from an experiment (h5 format). "
+            "The order of the traces in the dataset should correspond to "
+            "the order of masks in `roi_masks_path`.")
+    )
+    neuropil_traces_data_key = fields.Str(
+        required=False,
+        missing="data",
+        default="data",
+        description=("Key in `neuropil_traces_path` h5 file where data array "
+                     "is stored.")
+    )
+    traces_path = H5InputFile(
+        required=True,
+        description=(
+            "Path to traces extracted from an experiment (h5 format). "
+            "The order of the traces in the dataset should correspond to "
+            "the order of masks in `roi_masks_path`.")
+    )
+    traces_data_key = fields.Str(
+        required=False,
+        missing="data",
+        default="data",
+        description=("Key in `traces_path` h5 file where data array is "
+                     "stored.")
+    )
+    roi_masks_path = fields.InputFile(
+        required=True,
+        description=("Path to json file of segmented ROI masks. The file "
+                     "records must conform to the schema "
+                     "`RoiJsonRecordSchema`")
+    )
+    rig = fields.Str(
+        required=True,
+        description=("Name of the ophys rig used for the experiment.")
+    )
+    depth = fields.Int(
+        required=True,
+        description=("Imaging depth for the experiment.")
+    )
+    full_genotype = fields.Str(
+        required=True,
+        description=("Genotype of the experimental subject.")
+    )
+    targeted_structure = fields.Str(
+        required=True,
+        description=("Name of the brain structure targeted by imaging.")
+    )
+    classifier_model_path = fields.Str(
+        required=True,
+        description=("Path to model. Can either be an s3 location or a "
+                     "path on the local file system. The output of the model "
+                     "should be 0 if the ROI is classified as not a cell, "
+                     "and 1 if the ROI is classified as a cell.")
+    )
+    trace_sampling_fps = fields.Int(
+        required=True,
+        description="Sampling rate of trace (frames per second)."
+    )
+    downsample_to = fields.Int(
+        required=False,
+        default=4,
+        validate=lambda x: x > 0,
+        description=("Target rate to downsample trace data (frames per "
+                     "second). Will use average bin values for downsampling.")
+    )
+    output_json = fields.OutputFile(
```

Comment:
this is not necessary as it is already [here](https://github.com/AllenInstitute/argschema/blob/afc2b9241176a587e543f71830f2968d7a552cb7/argschema/schemas.py#L32-L44)
but, it doesn't hurt if you're just trying to be explicit.